packages/js/examples/react-audio/stories/components/Utilities.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useRef, useState } from 'react';
+import { TelnyxRTC } from '@telnyx/webrtc';
+
+function Utilities({ environment, username, password }) {
+  const clientRef = useRef();
+  const [isConnected, setIsConnected] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const initClient = () => {
+    clientRef.current = new TelnyxRTC({
+      env: environment,
+      login: username,
+      password,
+    });
+  };
+
+  const connect = async () => {
+    if (environment && username && password) {
+      if (isConnected) {
+        setIsConnected(false);
+        setMessage('Reconnecting...');
+
+        await clientRef.current.disconnect();
+      }
+
+      initClient();
+
+      clientRef.current.on('telnyx.ready', () => {
+        setIsConnected(true);
+        setMessage('Connected');
+      });
+
+      clientRef.current.on('telnyx.error', () => {
+        setMessage('Received error attempting to connect');
+      });
+
+      await clientRef.current.connect();
+
+      clientRef.current = clientRef.current;
+    } else {
+      setMessage('Username and Password are required');
+    }
+  };
+
+  const getAudioInDevices = async () => {
+    const results = await clientRef.current.getAudioInDevices();
+
+    setMessage(
+      `Audio input devices: ${
+        results.length ? results.map(({ label }) => label).join(', ') : 'None'
+      }`
+    );
+  };
+
+  const getAudioOutDevices = async () => {
+    const results = await clientRef.current.getAudioOutDevices();
+
+    setMessage(
+      `Audio output devices: ${
+        results.length ? results.map(({ label }) => label).join(', ') : 'None'
+      }`
+    );
+  };
+
+  if (!clientRef.current && environment && username && password) {
+    initClient();
```

Comment:
Why do you need this? I saw that you already call this function inside `connect()` line 26.

Reply:
`connect` is called when the button is clicked to re-initialize the client